packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2194,13 +2217,26 @@ class _RenderFocusTrap extends RenderProxyBoxWithHitTestBehavior {
     }
   }
 
+  /// The touch focus dropping behavior is only present on mobile browsers.
```

Comment:
What about laptop with a touch screen?  I imagine that should remove focus as well.

Reply:
Yes, that's right. I thought that was to make a slow pace compatible only with mobile. The web could work with desktop browsers inherently, though. I'm not sure but have a try.